src/Utils/getFormattedNumber.ts

```diff
@@ -61,18 +82,27 @@ export function getFormattedRoundedNumber(
   return formatter.format(number);
 }
 
-export function getFormattedNumber(
-  langCode: string,
-  number: number
-) {
+export function getFormattedNumber(langCode: string, number: number) {
   const formatter = new Intl.NumberFormat(langCode);
   return formatter.format(number);
 }
 
-export function parseNumber(
-  langCode: string,
-  number: number
-) {
+export function parseNumber(langCode: string, number: number) {
   const parser = new NumberParser(langCode);
   return parser.parse(number);
 }
+
+export function parseJwt(token: string) {
+  var base64Url = token.split(".")[1];
+  var base64 = base64Url.replace(/-/g, "+").replace(/_/g, "/");
+  var jsonPayload = decodeURIComponent(
+    atob(base64)
+      .split("")
+      .map(function (c) {
+        return "%" + ("00" + c.charCodeAt(0).toString(16)).slice(-2);
+      })
+      .join("")
+  );
+
+  return JSON.parse(jsonPayload);
+}
```

Comment:
I'll suggest moving this to a separate file as it's not related to number formatting in any way.

Reply:
As I wrote above I suggest rather using the same code @sarveshpro used at https://github.com/Plant-for-the-Planet-org/planet-donations/pull/227/files#r785306369 (maybe after @sarveshpro simplified it as I suggest in his PR).